conf/config.properties.example

```diff
@@ -322,9 +322,23 @@ server.delete.files.on.reload=false
 server.projects.type=HardCoded
 client.require.login=true
 
+# blueimp for local file storage
+# gcs for Google Cloud Storage
+# custom for custom file storage (add file service url below)
 server.service.external.file.type=blueimp
+server.service.external.file.uploadTempFiles=true
+server.migrateCmpdRegBulkLoaderFilesToSubfoldersOnStart=true
```

Comment:
Plan on leaving this as a default and then switching to false in a few quarters so its not run on every start of ACAS.